resources/migrations/000_migrations.yaml

```diff
@@ -15083,6 +15083,20 @@ databaseChangeLog:
             constraintName: fk_conn_impersonation_group_id
             onDelete: CASCADE
 
+  - changeSet:
+      id: v47.00-058
+      author: qnkhuat
+      comment: 'Drop parameter_card.entity_id'
+      changes:
+        - dropColumn:
+            tableName: parameter_card
+            columnName: entity_id
+      rollback:
+        - sql:
+            sql: >-
+                ALTER TABLE parameter_card
+                ADD COLUMN IF NOT EXISTS entity_id CHAR(21) NULL;
```

Comment:
in case someone ran the migration from `v48`, this column will be existed already